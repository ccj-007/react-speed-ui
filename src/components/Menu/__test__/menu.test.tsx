import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { Menu } from '../../index';

describe('test Menu component', () => {
  it('should render', () => {
    const { asFragment } = render(<Menu />);
    expect(asFragment()).toMatchSnapshot();
  });
});
