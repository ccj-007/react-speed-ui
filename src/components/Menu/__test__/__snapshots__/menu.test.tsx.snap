// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Menu component should render 1`] = `
<DocumentFragment>
  <ul
    class="viking-menu menu-horizontal"
    data-testid="test-menu"
  />
</DocumentFragment>
`;
